composer/spec/dependabot/composer/file_parser_spec.rb

```diff
@@ -49,6 +49,16 @@
       end
     end
 
+    context "null dependencies with lockfile" do
+      let(:project_name) { "null_dependencies_with_lockfile" }
+      let(:name) { "phpunit/phpunit" }
+      let(:type) { "development" }
+      describe "no dependencies" do
+        subject { dependencies }
+        its(:length) { is_expected.to be >= 0 }
```

Comment:
Would you mind adding a link to the bug/issue this is addressing somewhere? I understand what this test is doing but I'm not sure what behavior it's trying to fix/verify.

Reply:
Hey Bryan, 
The issue is https://github.com/dependabot/dependabot-core/issues/5919

Reply:
Thanks!